tests/contributing.py

```diff
@@ -268,7 +269,8 @@ def validate_transformer_code_style(transformer):
     Args:
         transformer (string or rdt.transformers.BaseTransformer):
             The transformer to validate.
-    Output:
+
+    Returns:
```

Comment:
I noticed some of the other validation functions say `Output:` instead of `Returns`. I know that's not necessarily tied to this PR, but since I think this is the last one that will touch this file, can we make them all consistent?